src/components/List/List.js

```diff
@@ -1,21 +1,94 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
```

Comment:
I think it is clearer why those exceptions are made if they are placed at the line where they are needed. That can also help with linting in the future, so that these exceptions are only added after per-case consideration.